wellregistry/registry/admin/monitoring_location.py

```diff
@@ -16,199 +16,202 @@
-		'tooltip': 'Is the well part of the QW network (default is \'Yes\')? All wells, marked as \'Yes\' will be\
+},
+    {
+    'form_field': 'qw_sn_flag',
+    'tooltip': 'Is the well part of the QW network (default is \'Yes\')? All wells, marked as \'Yes\' will be\
  included in the NGWMN QW network. (Yes/No)'
-	},
-	{
-		'form_field': 'qw_well_type',
-		'tooltip': 'Three choices are possible: (a) \'Trend\', (b) \'Surveillance\',\
+},
+    {
+    'form_field': 'qw_well_type',
+    'tooltip': 'Three choices are possible: (a) \'Trend\', (b) \'Surveillance\',\
  or (c) \'Special Studies\'. \'Trend\' wells have a monitoring frequency appropriate\
  to determine long-trends and seasonal variability (quarterly), \'Surveillance\' wells\
  are \'synoptic\' snapshots of data used to tied together the \'Trend\' wells. \'Special Studies\' wells\
  are likely to be local areas of depletion or impairment. (See the NGWMN Monitoring Categories Tip Sheet\
  for additional guidance)'
-	},
-	{
-		'form_field': 'qw_network_name',
-		'tooltip': 'The system from which water quality data from the well or spring will be served to the portal'
-	},
-	{
-		'form_field': 'qw_well_chars',
-		'tooltip': 'The characteristics of the aquifer that the well represents. There are 3 options:\
+},
+    {
+    'form_field': 'qw_network_name',
+    'tooltip': 'The system from which water quality data from the well or spring will be served to the portal'
+},
+    {
+    'form_field': 'qw_well_chars',
+    'tooltip': 'The characteristics of the aquifer that the well represents. There are 3 options:\
     (a) \'Background\', (b) \'Suspected/Anticipated Changes\', or (c) \'Known Changes\'.\
  This column is blank if the site is still in the baseline period. (See the NGWMN Subnetwork Tip Sheet for additional\
  guidance)'
-	},
-	{
-		'form_field': 'qw_baseline_flag',
-		'tooltip': 'A \'baseline\' period of at least 5 years of data must be available to achieve\
+},
+    {
+    'form_field': 'qw_baseline_flag',
+    'tooltip': 'A \'baseline\' period of at least 5 years of data must be available to achieve\
  the \'baseline period\' for a well or spring. Has the baseline period for water quality been satisfied\
  (are there 5 years of data)? (Yes/No) (See the NGWMN Subnetwork Tip Sheet for additional guidance)'
-	},
-	{
-		'form_field': 'qw_well_purpose',
-		'tooltip': 'A two-category classification to document well\'s original purpose:\
+},
+    {
+    'form_field': 'qw_well_purpose',
+    'tooltip': 'A two-category classification to document well\'s original purpose:\
  (a) \'Dedicated Monitoring/Observation\', or (b) \'Other\' (i.e. not a dedicated monitoring well)'
-	},
-	{
-		'form_field': 'link',
-		'tooltip': 'URL to a cooperators site or any other relevant site that contains additional information\
+},
+    {
+    'form_field': 'link',
+    'tooltip': 'URL to a cooperators site or any other relevant site that contains additional information\
  about the well'
-	}
+}
```

Comment:
Your IDE edited nearly 200 lines here. please use git to revert and BTW git has a -p to partially add and commit lines if you cannot stop the IDE from formatting.